index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>My Portfolio</title>
+</head>
+<body> 
+ <div class="navmenu">
+    <header class="logo">My logo</header>
+    <nav class="menu">
+    <div class="button">
+        <img src="Union.png" alt="hamburger">
+    </div>
+
+    </nav>
+</div>
+<div class="shapes">
+    <img src="header-shapesmobile@2x.png" alt="background-image">
+</div>
```

Comment:
- [ ] Decorations are always background images, not img tags. I would recommend implementing them in your CSS.